jest.config.ts

```diff
@@ -0,0 +1,8 @@
+import type { JestConfigWithTsJest } from 'ts-jest';
+
+const jestConfig: JestConfigWithTsJest = {
```

Comment:
Jest is really not made to work with typescript and is a pain to use with it. I would recommend you to use something else like Vitest which handles typescript natively.